tensorrt_llm/_torch/modules/fused_moe/fused_moe_cutlass.py

```
@@ -1495,26 +1501,28 @@ def _reducescatter_or_allreduce(x_, idx):
 
             outputs_list = []
             # Postpone reduce-scatter/all-reduce to the next iteration to achieve better overlap
-            for idx_chunk, (x, router_logits) in enumerate(
-                    zip(x_list, router_logits_list)):
+            for idx_chunk, (x, router_logits, input_ids) in enumerate(
+                    zip(x_list, router_logits_list, input_ids_list)):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)